docs/source/api/model_graph.rst

```diff
@@ -0,0 +1,7 @@
+Graphing Models
+-------------
```

Comment:
Don't these need to match the length of the text?

Reply:
Apparently not:

![image](https://user-images.githubusercontent.com/3274/58983339-3415c280-879c-11e9-9025-ef0ddf3f1824.png)